src/get-started/helloWorld/1-overview.md

```diff
@@ -60,7 +60,7 @@ description:
 </div>
 
 <div class="bg-gradient-to-r from-purple-400/10 to-purple-400/5 rounded-[6px] p-6 border-l-4 border-fuchsia-700 mb-6">
-  <p class="m-0!">This is where <span class="text-fuchsia-700 font-semibold">iExec</span> comes in! We provide tools to easily add <span class="text-fuchsia-700 font-semibold">privacy</span> and <span class="text-fuchsia-700 font-semibold">monetization</span> features into your iApp.</p>
+  <p class="m-0!">This is where <span class="text-fuchsia-700 font-semibold">iExec</span> comes in. iExec provides tools to easily add <span class="text-fuchsia-700 font-semibold">privacy</span> and <span class="text-fuchsia-700 font-semibold">monetization</span> features into your iApp.</p>
```

Comment:
Into your App
iApp = App running in the TEE
App = App for end user with an UX and the iExec Privacy feature